### PR TITLE
Add cert to rally config

### PR DESCRIPTION
**Pull request by srsakhamuri**

### CODE REVIEW

File: cookbooks/bcpc-extra/templates/default/rally.existing.json.erb [hunk 1]
```diff
@@ -20,4 +20,5 @@
         "project_domain_name": "<%=@domain_name%>"
     },
     "https_insecure": false,
+    "https_cacert": "/usr/local/share/ca-certificates/ssl-bcpc.crt",
```
Comment (justinjpacheco) [inline cookbooks/bcpc-extra/templates/default/rally.existing.json.erb]: we should probably make this file path a variable in attributes/default.rb as it is referenced directly in multiple locations. but we should probably do that in a separate commit

Comment (srsakhamuri) [reply cookbooks/bcpc-extra/templates/default/rally.existing.json.erb]: Agree
